src/main/kotlin/com/exactpro/th2/rptdataprovider/entities/filters/events/AttachedMessageFilter.kt

```diff
@@ -39,6 +40,7 @@ class AttachedMessageFilter(
             return AttachedMessageFilter(
                 negative = filterRequest.isNegative(),
                 conjunct = filterRequest.isConjunct(),
+                strict = filterRequest.isStrict(),

```

Comment:
Please don't use strict parameter in this filter (because match by id subsequence is not needed)